src/content/docs/release-notes/synthetics-release-notes/containerized-private-minions-release-notes/containerized-private-minion-cpm-3040.mdx

```diff
@@ -0,0 +1,10 @@
+---
+subject: Containerized private minion (CPM)
+releaseDate: '2021-03-18'
+version: 3.0.40
+---
+
+### Fixes
+
+* Sticky bit is now correctly assigned for /tmp mounted volume on Kubernetes deployment
```

Comment:
I'm not sure I understand this sentence, @cleon-nr. Could you clarify it?

Reply:
This is in reference to a Linux permission option that's being set on a mounted `/tmp` directory when the CPM is deployed on Kubernetes. We have an initContainer that recursively changes some permissions on these mounted subpaths and the "sticky bit" wasn't being applied. So going from `777` to `1777` is mainly what this is referring to. The `1` referring to the "sticky bit" as described below.

> The restricted deletion flag or sticky bit is a single bit, whose interpretation depends on the file type. For directories, it
> prevents unprivileged users from removing or renaming a file in the directory unless they own the file or the directory; this is called the restricted deletion flag for the directory, and is commonly found on world-writable directories like /tmp. For regular files on some older systems, the bit saves the program's text image on the swap device so it will load more quickly when run; this is called the sticky bit.